backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/ConfirmationCodesController.java

```diff
@@ -31,7 +27,7 @@
 @Tag(name = "Confirmation codes", description = "Endpoint for managing user confirmation code resources.")
 public class ConfirmationCodesController {
 
-	private final ConfirmationCodeModelMapper confirmationCodeModelMapper = Mappers.getMapper(ConfirmationCodeModelMapper.class);
```

Comment:
Can `ConfirmationCodeModelMapper` now be removed as well since it's not being used by this controller?